browser/src/components/core/AutoGrowInput.tsx

```diff
@@ -45,18 +46,25 @@ export function AutoGrowInput({
     //   >
     //     {value}
     //   </span>
-    <div className={`grow-wrap ${className ? className : ""}`}>
-      {/* <div className={`grow-wrap`}> */}
-      <textarea
-        {...(extraProps || {})}
-        className="form-textarea mt-1 block w-full"
-        style={{ width: "300px", minHeight: "200px" }}
-        // placeholder="Enter your response to the prompt..."
-        value={value}
-        onChange={(event) => onChange(event.target.value)}
-      ></textarea>
-    </div>
-
+    <>
+      <div className={`grow-wrap ${className ? className : ""}`}>
+        {/* <div className={`grow-wrap`}>
+        <textarea
+          {...(extraProps || {})}
+          className="form-textarea mt-1 block w-full"
+          style={{ width: "300px", minHeight: "200px" }}
+          // placeholder="Enter your response to the prompt..."
+          // value={value}
+          // onChange={(event) => onChange(event.target.value)}
+        >
+        </textarea> */}
+        <Editor
```

Comment:
also q on behavior, can you try adding a long thing in and seeing if the editor box auto-grows? don't need to address in this PR but would be a nice follow-up to have it auto expand with more content

Reply:
yeah makes sense considering its called AutoGrowInput haha. I tested live behavior and it doesn't seem to autogrow, so based this off that, is that intended?